app/controllers/admin/orders_controller.rb

```diff
@@ -1,8 +1,17 @@
 class Admin::OrdersController < OrdersController
   before_action :require_admin
-    def update
-      @order = Order.find(params[:id])
-      @order.ship
-      redirect_to admin_path
-    end
+
+  def update
+    @order = Order.find(params[:id])
+    @order.ship
+    redirect_to admin_path
+  end
+
+  def index
+    @user = User.find(params[:user_id])
+  end
+
+  def show
+    @order = Order.find(params[:id])
+  end
```

Comment:
This show method already exists in the OrdersController this class is inheriting from.